bindings/python/iio.py

```diff
@@ -436,7 +475,20 @@ def _Attr__write(self, value):
 		_d_write_debug_attr(self._device, self._name_ascii, value.encode('ascii'))
 
 class Channel(object):
+
```

Comment:
I think something strange is happening with the Codacy check. I have removed the blank line from here but the error persists. All other classes have the same format (with a blank line before the docstring), but Codacy doesn't report errors from there. 
More than this, most modifications of this pull request are adding a blank line before the docstring (because Codacy was complaining about it before).

Reply:
Ok - will merge and see what happens... Thanks for checking.